modules/local/align_linktable/main.nf

```diff
@@ -0,0 +1,69 @@
+process ALIGNMENT_LINKTABLE {
+
+    /*
+    This module aligns reads to the reference using specified arguments and produces output in the form of FASTQ.GZ, BAM and BAM.BAI files.
+    It also runs:
+     - eukcc linktable
+    */
+
+    label 'process_medium'
+
+    tag "${meta.id} align to ${ref_fasta}"
+
+    container 'quay.io/microbiome-informatics/bwa_eukcc:2.2.1_2.0'
+
+    input:
+    tuple val(meta), path(reads), path(ref_fasta), path(bins, stageAs: "bins/*")
+    val(binner)
+
+    output:
+    tuple val(meta), path("*.links.csv"), emit: links_table
+    path "versions.yml"                 , emit: versions
+
+    script:
+
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def samtools_args = task.ext.alignment_args
+
+    """
+    mkdir -p output
+
+    echo " ---> index fasta"
+    bwa-mem2 index ${ref_fasta}
+
+    echo " ---> mapping files to host genome"
```

Comment:
Again, here you are " ---> mapping reads to assembly"
Otherwise it can be confused with a decontamination step